cypress/e2e/tests/migration/applicationinventory/applications/interlinked.test.ts

```diff
@@ -148,6 +149,8 @@ describe(["@tier3"], "Applications interlinked to tags and business service", ()
         //Verify that values show blank
         cy.get(stakeholderSelect).should("have.value", "");
         cy.get(stakeholdergroupsSelect).should("have.value", "");
+        clickByText(button, "Cancel");
+        clickByText(button, "Continue");
```

Comment:
I see uses of Cancel and Continue in our test cases, for future we can add in these in Constant too